insights/client/data_collector.py

```diff
@@ -39,31 +39,24 @@ def _process_content_redaction(filepath, exclude, regex=False):
     /etc/insights-client/.exp.sed and and the contents of "exclude"
```

Comment:
It is no based in that file anymore.

Reply:
So I made an incorrect understanding, I thought you (the client team) might want to keep these `settings` until remove this file from the RPM package.

I'll remove them in the next commit soon.

Reply:
Sure, I'll update this docstring. Thanks for reviewing.